main.py

```diff
@@ -0,0 +1,82 @@
+import time
+import argparse
+from os.path import join, dirname
+
+from common import (get_dict_from_yaml, JUMP_HOST_CONFIG_PATH, JUMP_HOST_DIR,
+                    JUMP_HOST_LICENSE_PATH, JUMP_HOST_SSH_KEY_PATH,
+                    OLD_JUMP_HOST_DIR, logger, VM)
+
+INSTALL_CLUSTER_SCRIPT = 'install_cluster.py'
+
+
+def _prepare_jump_host(jump_host, license_path, config_path):
+    commands_list = [
+        'sudo yum install -y epel-release',
```

Comment:
While epel-release probably needs to be done as its own command, it'd be best to get the rest into as few calls as possible- ideally one yum and one pip if we can.